impl/doc/display.md

```diff
@@ -176,3 +234,10 @@ assert_eq!(UnitStruct {}.to_string(), "UnitStruct");
 assert_eq!(PositiveOrNegative { x: 1 }.to_string(), "Positive");
 assert_eq!(PositiveOrNegative { x: -1 }.to_string(), "Negative");
 ```
+
+
+
+
```

Comment:
were all these newlines intentional?

Reply:
@JelteF yes.... 4 newlines before `##` headers and 2 newlines before `###` headers.